omp/test/matrix/batch_ell_kernels.cpp

```diff
@@ -68,8 +68,10 @@ class BatchEll : public ::testing::Test {
           mtx_size(10, gko::dim<2>(12, 7)),
           rand_engine(42)
     {
-        mtx = Mtx::create(exec, 2, gko::dim<2>{2, 3}, 2);
-        mtx2 = Mtx::create(exec, 2, gko::dim<2>{3, 3}, 2);
+        mtx = Mtx::create(exec, gko::batch_dim<2>{2, gko::dim<2>{2, 3}},
+                          gko::batch_stride{2, 2});
```

Comment:
This is more verbose. Maybe we should have an implicit conversion from the previous version ? Not sure how error-proof that would be.